src/scripts/main.js

```diff
@@ -1,7 +1,24 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const blockEl = document.createElement('div');
+  const titleEl = document.createElement('h2');
+  const textEl = document.createElement('p');
+
+  blockEl.classList = `notification ${type}`;
```

Comment:
The use of `classList` to assign a string of classes is incorrect. You should use `className` instead. `classList` is used for adding, removing, or toggling individual classes.